src/GuiBase/Timeline/HelpDialog.ui

```diff
@@ -0,0 +1,1004 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Reading a .ui file is something that you can't expect for a PR reviewer. Recall that such a file is generated by Qt Designer and that only a few hardcore developers will open and modify it outside of this tool.
And having hard-coded keyboard/mouse keymapping (hidden in this file, e.g. line180 for the "Go to start" shortcut" ) is clearly not a good idea as Radium might be used on several systems/keyboard configurations. 
Why not using KeyMappingManager to do that and have your dialog print what is stored on the manager instead of displaying hard-coded information ?